core-java-modules/core-java/pom.xml

```diff
@@ -61,6 +61,11 @@
             <artifactId>moneta</artifactId>
             <version>${javamoney.moneta.version}</version>
         </dependency>
+        <dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-core</artifactId>
+			<version>${spring.core.version}</version>
+		</dependency>
```

Comment:
Please indent with spaces like the rest of the file.

Reply:
Done